src/notyf.ts

```diff
@@ -47,6 +47,16 @@ export default class Notyf {
     return this.open(options);
   }
 
+  public info(payload: string | Partial<INotyfNotificationOptions>) {
```

Comment:
Can you please add tests for these new features?

Reply:
I will try to add tests. I am a still learning how to code...